From e968ce4d619201994330953b647b75c5f9a92a27 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 10 Jul 2019 20:24:54 +0200
Subject: [PATCH 1/1] mailmap: provide usage instruction

Looking at the contents of file .mailmap it seems that some editors assumed
that translation is done by entering multiple lines into the file and the
last one replaces the others. This is not how it works. The translation
occurs according to entries in single lines as described in the
git-check-mailmap man-page.

Add a description of the file format.

Add an entry for Alexander Graf as his old email address is not valid
anymore.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
Resent, CC: U-Boot Mailing List <u-boot@lists.denx.de>
---
 .mailmap | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/.mailmap b/.mailmap
index 1bee048d13..dc72f24688 100644
--- a/.mailmap
+++ b/.mailmap
@@ -4,10 +4,18 @@
 # and/or not always written the same way, making contributions from the
 # same person appearing not to be so or badly displayed.
 #
+# This file is also used by scripts/get_maintainer.pl.
+#
 # This file can be modified by hand or updated by the following command:
 #  scripts/mailmapper > tmp; mv tmp .mailmap
 #
+# Entries in this file take one of the following forms:
+# Proper Name <commit@email.xx>
+# <proper@email.xx> <commit@email.xx>
+# Proper Name <proper@email.xx> <commit@email.xx>
+# Proper Name <proper@email.xx> Commit Name <commit@email.xx>
 
+Alexander Graf <agraf@csgraf.de> <agraf@suse.de>
 Allen Martin <amartin@nvidia.com>
 Andreas Bießmann <andreas.devel@googlemail.com>
 Andreas Bießmann <andreas@biessmann.org>
-- 
2.20.1

